cdc_shell.c

```diff
@@ -394,6 +762,33 @@ static const cdc_shell_cmd_t cdc_shell_commands[] = {
                           "Example: \"uart 1 tx output od\" sets UART1 TX output type to open-drain\r\n"
                           "Example: \"uart 3 rts active high dcd active high pull down\" allows to set multiple parameters at once.",
     },
+    {
```

Comment:
How about split cdc_shell.c for sub-modules (like uart, gpio, config, etc.) and format this table from compiler sections, to make this project easilly expandable.